_posts/multiworld/2024-03-10-multiworld-with-nintendont.md

```diff
@@ -0,0 +1,79 @@
+      * Third-party adapters Ethernet using the ASIX AX88772 or AX88772A chipsets can work too, such as [this one](https://www.amazon.com/gp/product/B00MYT47EA/).  
+  * **A Supported Controller**  
+    * For consoles with built-in GC controller ports (most Wiis: RVL-001):  
+      using those ports with a GameCube controller is recommended.  
+    * For consoles without built-in GC controller ports (all Wii Us, some Wiis: RVL-101, RVL-201):  
+      using a GameCube Controller Adapter with a GameCube controller is recommended:  
+      Both the [official Nintendo adapter](https://smashbros.nintendo.com/buy/accessories/) and the [Mayflash adapter (in Wii U/Switch mode)](https://www.amazon.com/Mayflash-GameCube-Controller-Adapter-Switch/dp/B00RSXRLUE/) will work.  
+    * Otherwise, see the [list of Nintendont-supported controllers](https://gbatemp.net/threads/nintendont.349258/#Controllers)  
+  
+## Steps ##  
+  
+ * Insert the SD Card into your computer.  
+ * Generate and export the game ISO.  
+ * Copy the game ISO to the `games` folder of the root of your SD Card.  
+   * If the `games` folder is not present, create it.  
+     For example, on Windows: *`S:\games\Prime Randomizer - MyMultiworldSession  - Nystrata.iso`*  
+ * Insert the SD Card into the console.  
+ * Start the Homebrew Channel  
+ * Press HOME or START/PAUSE to open the Hombrew Channel Menu.  
+   Take note of the IP Address in the top-left corner.  
+   * If it doesn't appear, make sure your console's Internet settings are configured correctly.  
+ * Open the Games Connections menu:  
+   * If in the main Randovania window:  
+     * *Open* -> *Automatic Item Tracker* -> *Connections*  
+   * If in the Multiworld Session window:  
+     * *Session and Connectivity* tab -> *Edit Game Connections*  
+ * Click *Add new connection* and select *Nintendont*  
+ * Enter the IP address noted from the console earlier.  
+ * Find the newly created Nintendont entry, then click the ***`...`*** button next to it to reveal a menu.  
+ * Click *Upload to Nintentdont to Homebrew Channel*  
+   This will upload the app the Homebrew Channel and launch it automatically.  
+ * With Nintendont, when asked to select a storage device, select SD  
+ * <ins>**If you have a real GameCube Memory Card that you'd like to use:**</ins>  
+    * Skip over the next step and select your game.  
+ * <ins>**If you do not want/cannot use a GameCube Memory Card:**</ins>  
+    * While in the game selection screen, press **B** to enter Nintendont settings.  
+    * Set *Memcard Emulation* to `ON`.  
+    * Then press **B** again to return to the game list.  
+ * Then select your game.  
+   The game will start when a connection is made to Randovania.  
+   * If the screen remains black after a long moment, it may be that it's failing to connect to Randovania:  
```

Comment:
as I mentioned earlier, this is typically due to unstable wifi connections. you can either just retry until it works, or try a wired connection